packages/fields/types/CalendarDay/views/Cell/Cell.js

```diff
@@ -1,3 +1,5 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
Maybe we should use [ESLint's overrides option](https://eslint.org/docs/user-guide/configuring#disabling-rules-only-for-a-group-of-files) to disable the rule for all the field type views? Using preconstruct also essentially has the same effect as using this rule since it'll throw an error if any node module is imported that isn't specified in its packages' dependencies or peerDependencies

Reply:
I'd actually rather leave those comments in-place until we decide whether we will disable the rule permanently for these files and depend on preconstruct's errors, or work out a way to make the rule compatible with how preconstruct does entrypoints.

They're a bit in-your-face, and I think that's a good thing.